tests/run.sh

```diff
@@ -85,8 +85,11 @@ cd $build_dir
 AWESOME_OPTIONS="$AWESOME_OPTIONS --search lib/"
 # Add test dir (for _runner.lua).
 AWESOME_OPTIONS="$AWESOME_OPTIONS --search $this_dir"
-XDG_CONFIG_HOME="./"
-export XDG_CONFIG_HOME
+export XDG_CONFIG_HOME="./"
+
+# Set LUA_PATH for codecov.
+# (https://github.com/awesomeWM/awesome/issues/1049#issuecomment-247876195)
+export LUA_PATH="$(lua -e 'print(package.path)');lib/?.lua;lib/?/init.lua"
```

Comment:
Unrelated, but I just stumbled upon the following:

> Any ";;" in the value of the environment variable is replaced by the default path. 

https://www.lua.org/manual/5.1/manual.html#pdf-LUA_PATH

I think we should make use of that. :-)